kafo.gemspec

```diff
@@ -23,11 +23,14 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov', '< 0.9'
   spec.add_development_dependency 'ci_reporter', '~> 1.9.0'
 
+  spec.add_development_dependency 'puppet'
+  # this is required if puppet is greater than 4.0
+  # spec.add_development_dependency 'puppet-strings'
```

Comment:
if I add it there, jenkins is unhappy that we specify puppet twice (it adds it to Gemfile too). Maybe we could move the logic into Gemfile - for puppet >= 4 install puppet-strings, but I'm not sure if we can detect current puppet version in the middle of Gemfile, it might not be installed by bundler yet